app/DevelopmentConfig.hs

```diff
@@ -109,45 +191,42 @@ createReqBody org = object [ "campuses" .= ([] :: [T.Text]),
                         "dayPreferences" .= ([] :: [T.Text]),
                         "deliveryModes" .= ([] :: [T.Text]),
                         "departmentProps" .= ([] :: [T.Text]),
-                        "direction" .= ("asc" :: T.Text),
-                        "divisions" .= [T.pack "ARTSC"],
+                        "direction" .= directionValue config,
+                        "divisions" .= divisionsValue config,
                         "instructor" .= ("" :: T.Text),
-                        "page" .= (1 :: Int),
-                        "pageSize" .= (200 :: Int),
+                        "page" .= pageValue config,
+                        "pageSize" .= pageSizeValue config,
                         "requirementProps" .= ([] :: [T.Text]),
-                        "sessions" .= [T.pack "20249", T.pack "20251", T.pack "20249-20251"],
+                        "sessions" .= sessionsValue config,
                         "timePreferences" .= ([] :: [T.Text])
                      ]
 
 -- | The headers for the HTTP request
-reqHeaders :: RequestHeaders
-reqHeaders = [("Content-Type", "application/json"), ("Accept", "application/json")]
+reqHeaders :: IO RequestHeaders
+reqHeaders = do reqHeadersValue <$> loadConfig
```

Comment:
This also shouldn't be a configuration value; it's hard-coded based to match the implementation of our web parser.

Removing this has the side benefit of eliminating the need for the `HeaderPair` type definition and the `case-insensitive` library.